### PR TITLE
[Mobizel][MarkdownDocsBundle] Add recipe for v0.9 release

### PR DESCRIPTION
**Pull request by loic425**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by loic425** [comment]
@Nyholm Could you review this small one please?

**Comment by loic425** [comment]
@Nyholm small ping

File: mobizel/markdown-docs-bundle/0.1/docs/index.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # Documentation
```
Comment (Nyholm) [inline mobizel/markdown-docs-bundle/0.1/docs/index.md:1]: Why do I, as a user, need this file?

Comment (loic425) [reply mobizel/markdown-docs-bundle/0.1/docs/index.md:1]: This is the homepage of your documentation.
If you don't have it, you have a not found. With this first markdown file you can check everything is setup correctly and I think it's important for the DX part.

Comment (Nyholm) [reply mobizel/markdown-docs-bundle/0.1/docs/index.md:1]: Okey. I thought it was just for documentation. 

Thank you for clarifying.

File: mobizel/markdown-docs-bundle/0.9/docs/index.md [hunk 2]
```diff
@@ -0,0 +1 @@
+../../0.1/docs/index.md
```
[the remark this answers is not in the document]
Comment (loic425) [reply mobizel/markdown-docs-bundle/0.9/docs/index.md:1]: Ok, I didn't know that.

**Review by Nyholm — APPROVED** [review]
Thank you

**Comment by loic425** [comment]
> Thank you

I thank you too 👍

**Pull request merged**